src/path/render.js

```diff
@@ -19,6 +32,17 @@ module.exports = function(g, path, l, t) {
 
   for (var i=0, len=path.length; i<len; ++i) {
     current = path[i];
+    // evaluate handlebar equations
+    for (var j=1; j < current.length; j++) {
+      // If command is enclosed in handlebars, evaluate
+      if (typeof current[j] === 'string' && current[j].startsWith('{{') && current[j].endsWith('}}')) {
+        current[j] = eval(current[j]);
```

Comment:
We should never make `eval` calls particularly when, as in this case, we're not thoroughly checking+sanitizing the contents of the handlebars. Instead, use vega-expression to parse the contents of each of the handlebars and return a JavaScript function that you can invoke directly rather than calling `eval`. Here's how I recommend going about it: 

1. Make vega-scenegraph dependent on vega-expression (i.e., `npm install -D vega-expression`). 
2. Add an optional argument (`expr`) to the parse function above. This argument receives the expression parser from Vega (`model.expr`) and needs to be passed down through the `View` and `Renderers`. If no `expr` argument is provided, create a new one from vega-expression. This way, vega-scenegraph is still standalone but, if invoked from Vega, will contain the custom functions we add there (e.g., `scale`, `iscale`, etc.).
3. Once you have an `expr` argument, you can invoke it like `expr(str)` where `str` is the string within the handlebars. Doing so returns an object with a function (`fn`) and some metadata about the expression. 
4. Then, in this `render` function you can call the `fn` rather than `eval`. 

Feel free to ping me with any questions. @RussellSprouts also has a lot of knowledge of how vega-expression works and can be invoked. That way, you don't need to wait on me to respond given the timezone differences.